src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -80,14 +81,28 @@ public static Address parseAddress(String address) throws ParseException {
         }
         return new Address(trimmedAddress);
     }
+
     /**
      * Parses a {@code String address} into an {@code Remark}.
      * Leading and trailing whitespaces will be trimmed.
      */
     public static Remark parseRemark(String remark) {
         requireNonNull(remark);
-        String trimmedAddress = remark.trim();
-        return new Remark(trimmedAddress);
+        String trimmedRemark = remark.trim();
+        return new Remark(trimmedRemark);
+    }
+
+    /**
+     * Parses a {@code String address} into an {@code Birthday}.
+     * Leading and trailing whitespaces will be trimmed.
+     */
+    public static Birthday parseBirthday(String birthday) throws ParseException {
+        requireNonNull(birthday);
+        String trimmedBirthday = birthday.trim();
+        if (!Birthday.isValidBirthday(trimmedBirthday)) {
```

Comment:
Respect the hussle but LocalDate::parse can cover the edge cases you missed in the regex, eg. 
2000-02-31 will pass isValidBirthday

Reply:
Oh yeaa true oops. I will come back and thoroughly change it to remove regex, but for now I just strengthen the validation condition using LocalDate